packages/widgets-manager/src/grid.ts

```diff
@@ -1,6 +1,68 @@
 import { makePostionStrategy } from './strategies/position/creator';
 import { App, Grid, GridCell, GridSettings, AddAppsStrategy, PositionStrategy } from "./types";
 
+export const gridRules = {
+  small: {
+    columns: 1,
+    gutter: 10,
+    padding: 10,
+    positions: [
+      'top-left',
+      'top-center',
+      'top-right',
+      'mid-left',
+      'mid-center',
+      'mid-right',
+      'bottom-left',
+      'bottom-center',
+      'bottom-right',
+    ]
```

Comment:
If "small" has one column, why are there positions for three columns?

Reply:
That it's because for a small case the script it's creating all the position in the same columns, therefore its a bug but because we don't want to change the API of the clients was easily maintaining the naming

Reply:
OK, gracias!